src/listformatter.cpp

```diff
@@ -73,19 +71,12 @@ std::string ListFormatter::format_list() const
 {
 	std::string format_cache = "{list";
 	for (const auto& line : lines) {
-		std::string str = line.first;
+		std::string str = line;
```

Comment:
I think this is just a leftover from refactoring, and you can remove it by changing the loop to `for (auto str : lines)`.

Reply:
Done: https://github.com/newsboat/newsboat/pull/1132/commits/74a00c75816e30a35de004e2359c31501f440944